index.js

```diff
@@ -23,7 +23,7 @@ function asyncDone(fn, cb){
   function done(){
     d.removeListener('error', onError);
     d.exit();
-    return cb.apply(null, arguments);
+    return cb.apply(cb, arguments);
```

Comment:
why is callback being applied as the context?  it doesn't seem that would effect anything.